helix-term/src/main.rs

```diff
@@ -91,8 +92,39 @@ FLAGS:
 
     setup_logging(logpath, args.verbosity).context("failed to initialize logging")?;
 
+    let themes = std::sync::Arc::new(Themes::load());
+
+    // initialize language registry
+    use helix_core::syntax;
+
+    // load $HOME/.config/helix/languages.toml, fallback to default config
+    let lang_config = std::fs::read(conf_dir.join("languages.toml"));
+    let toml = lang_config
+        .as_deref()
+        .unwrap_or(include_bytes!("../../languages.toml"));
+
+    let app_config = application::Configuration { theme: None };
+    let app_config = std::fs::read(conf_dir.join("config.toml"))
+        .map(|v| toml::from_slice(v.as_slice()))
+        .unwrap_or_else(|_| Ok(app_config.clone()))
```

Comment:
I don't quite follow the double unwrap. I think the `.map` on the previous line should be a `and_then`